src/atomate2/common/flows/phonons.py

```diff
@@ -0,0 +1,357 @@
+        Set to ``None`` to skip the
+        static energy computation
+    born_maker: .BaseAimsMaker or None
+        Maker to compute the BORN charges.
+    phonon_displacement_maker : .BaseAimsMaker or None
+        Maker used to compute the forces for a supercell.
+    generate_frequencies_eigenvectors_kwargs : dict
+        Keyword arguments passed to :obj:`generate_frequencies_eigenvectors`.
+    create_thermal_displacements: bool
+        Bool that determines if thermal_displacement_matrices are computed
+    kpath_scheme: str
+        scheme to generate kpoints. Please be aware that
+        you can only use seekpath with any kind of cell
+        Otherwise, please use the standard primitive structure
+        Available schemes are:
+        "seekpath", "hinuma", "setyawan_curtarolo", "latimer_munro".
+        "seekpath" and "hinuma" are the same definition but
+        seekpath can be used with any kind of unit cell as
+        it relies on phonopy to handle the relationship
+        to the primitive cell and not pymatgen
+    code: str
+        determines the dft code. currently only aims is implemented.
+        This keyword might enable the implementation of other codes
+        in the future
+    store_force_constants: bool
+        if True, force constants will be stored
+    socket: bool
+        If True, use the socket for the calculation
+    """
+
+    name: str = "phonon"
+    sym_reduce: bool = True
+    symprec: float = 1e-4
+    displacement: float = 0.01
+    min_length: float | None = 20.0
+    prefer_90_degrees: bool = True
+    get_supercell_size_kwargs: dict = field(default_factory=dict)
+    use_symmetrized_structure: str | None = None
+    bulk_relax_maker: BaseVaspMaker | BaseAimsMaker | None = None
+    static_energy_maker: BaseVaspMaker | BaseAimsMaker | None = None
+    born_maker: BaseVaspMaker | None = None
+    phonon_displacement_maker: BaseVaspMaker | BaseAimsMaker = None
+    create_thermal_displacements: bool = True
+    generate_frequencies_eigenvectors_kwargs: dict = field(default_factory=dict)
+    kpath_scheme: str = "seekpath"
+    code: str = None
+    store_force_constants: bool = True
+    socket: bool = False
+
+    def make(
+        self,
+        structure: Structure | MSONableAtoms,
+        prev_dir: str | Path | None = None,
+        born: list[Matrix3D] | None = None,
+        epsilon_static: Matrix3D | None = None,
+        total_dft_energy_per_formula_unit: float | None = None,
+        supercell_matrix: Matrix3D | None = None,
+    ):
+        """
+        Make flow to calculate the phonon properties.
+
+        Parameters
+        ----------
+        structure : .Structure
+            A pymatgen structure. Please start with a structure
+            that is nearly fully optimized as the internal optimizers
+            have very strict settings!
+        prev_dir : str or Path or None
+            A previous vasp calculation directory to use for copying outputs.
```

Comment:
Update necessary on the documentation

Reply:
Corrected and will do another round of docstring checks once the force fields are included